tuf/repository_lib.py

```diff
@@ -1520,14 +1573,22 @@ def generate_timestamp_metadata(snapshot_filename, version, expiration_date,
   tuf.formats.METADATAVERSION_SCHEMA.check_match(version)
   securesystemslib.formats.ISO8601_DATETIME_SCHEMA.check_match(expiration_date)
   securesystemslib.formats.NAME_SCHEMA.check_match(repository_name)
+  securesystemslib.formats.BOOLEAN_SCHEMA.check_match(use_length)
+  securesystemslib.formats.BOOLEAN_SCHEMA.check_match(use_hashes)
 
-  # Retrieve the versioninfo of the Snapshot metadata file.
   snapshot_fileinfo = {}
+
   length, hashes = securesystemslib.util.get_file_details(snapshot_filename,
-      tuf.settings.FILE_HASH_ALGORITHMS)
+    tuf.settings.FILE_HASH_ALGORITHMS)
```

Comment:
secure-systems-lab coding style is to double-indent continuation lines. Please restore indentation here.